config/routes.rb

```diff
@@ -11,7 +11,11 @@
 
   resources :recipes do
     resources :recipe_ingredients, only: [:edit, :update]
+    resources :reviews, only: [:new, :create]
   end
+
+  resources :reviews, only: [:show, :edit, :update, :destroy]
+
```

Comment:
I think show and edit are also nested under recipe :)

Reply:
fixed it!